### ISSUE TITLE
Wrong "footstep" audio is used when walking/running on the "Bridge".

### ISSUE THREAD

**Issue by Quintus24** [AmProsius/gothic-1-community-patch#301]
Hello, everyone!

I would like to report the following bug:

**Describe the bug:**
This issue can be encountered on the **Bridge** that connects to the **Goblin Caves**.
When crossing the bridge, running or walking on it, it can be noticed that a "**grass**/**dirt**" footstep sound is played instead of a "**wooden**" sound.

**Expected behavior:**
The correct "**wooden**" footstep sound plays when crossing the bridge.

**Steps to reproduce the issue:**
1. Travel to the **LOCATION_29_02** waypoint.
2. Cross the **Bridge** and listen carefully to the footstep sounds.
3. Notice the **grass**/**dirt**" footstep sound is used.

**Additional context**
Very minor issue, easily unnoticeable. 
Recommend lowering the music volume before testing.
I crossed that bridge several times until I finally noticed the sound it makes.

_Attaching screenshots for more clear information_.

1. Bridge location at **LOCATION_29_02** waypoint.
![Bridge Location @ LOCATION_29_02](https://user-images.githubusercontent.com/93062373/150651550-0b01b766-16f6-40f2-a9ed-e2efa345c5a8.JPG)

2. Overview of the **Goblin Cave** bridge.
![The Goblin Cave Bridge](https://user-images.githubusercontent.com/93062373/150651565-342cf763-6cf6-4ff3-9d5e-2ee41e90f604.JPG)

Thank You and Best Regards!
Quintus24

**Comment by AmProsius** [comment]
@szapp is this something we can fix programmatically?

**Comment by szapp** [comment]
Yes, it should be possible. I had started to develop the means to identify specific polygons/vertices and their associated material in #56. It would still require some research on how to adjust the properties reliably without jeopardizing other polygons assigned with the same material.
